README.md

```diff
@@ -161,3 +161,15 @@ The above scripts run the app using Debug configuration and for MindLogger's `d
 Common Public Attribution License Version 1.0 (CPAL-1.0)
 
 Refer to [LICENSE.md](./LICENSE.md)
+
+## Patch case your enviroment doesn't work on IOS
+
+Incluse #include <functional> on xplat/Flipper/FlipperTransportTypes.h
+https://github.com/facebook/flipper/commit/b3dcdb87f930dbbc9dbacb53ad60996e0111e7d8
+
+Replace the parameter type YGNodeRef with YGNodeConstRef in Pods/YogaKit/YGLayout.m:327
+https://github.com/facebook/react-native/issues/43383#issuecomment-1986232027
```

Comment:
You don't need either of those. Please use `bundler` when working with Cocoapods: https://omurbilgili.medium.com/unveiling-the-power-of-metro-bundler-in-react-native-bf68d151f063

Reply:
this was a patch suggested by @ChaconC .
I mentioned in documentation "in case IOS doesn't work".
@ChaconC  Can you share your point of view ?